src/helpers/Constants.js

```diff
@@ -40,6 +40,8 @@ export let screenWidth = width;
 export let screenHeight = height;
 export let isSmallScreen = screenWidth <= 340;
 export let isShortScreen = screenHeight <= 600;
+export const ratio = screenWidth < screenHeight ? screenHeight / screenWidth : screenWidth / screenHeight;
```

Comment:
Why are you exporting ratio?

Reply:
I don't need it exported, but thought it might be useful.
Do you want me to remove that? - Or maybe change the name?

Reply:
Let's remove the export till we need it and change the name to be more descriptive.

Reply:
Done